ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -26,38 +26,89 @@ import * as InputHelpers from '../input_helpers';
 //                                 RAID BUFFS
 ///////////////////////////////////////////////////////////////////////////
 
-// TODO: Classic buff icon by level
 export const AllStatsBuff = withLabel(
-	makeTristateRaidBuffInput({actionId: ActionId.fromSpellId(9885), impId: ActionId.fromSpellId(17055), fieldName: 'giftOfTheWild'}),
+	makeTristateRaidBuffInput({
+		actionId: (player) => player.getMatchingSpellActionId([
+			[1126, 1, 9],
+			[5232, 10, 19],
+			[6756, 20, 29],
+			[5234, 30, 39],
+			[8907, 40, 49],
+			[9884, 50, 59],
+			[9885, 60]
+		]), impId: ActionId.fromSpellId(17055), fieldName: 'giftOfTheWild'}),
```

Comment:
Here are some examples of how this is used to conditionally display the correct buff ranks